gradle/libs.versions.toml

```diff
@@ -280,6 +284,9 @@ androidx-test-runner = { module = "androidx.test:runner", version.ref = "android
 androidx-test-uiAutomator = { module = "androidx.test.uiautomator:uiautomator", version.ref = "androidx-test-uiAutomator" }
 androidx-test-work = { module = "androidx.work:work-testing", version.ref = "androidx-workManager" }
 androidx-benchmark-macro-junit4 = { group = "androidx.benchmark", name = "benchmark-macro-junit4", version.ref = "benchmark-macro-junit4" }
+zxing-core = { module = "com.google.zxing:core", version.ref = "zxing" }
+zxing-android-embedded = { module = "com.journeyapps:zxing-android-embedded", version.ref = "zxingAndroid" }
+gson = { module = "com.google.code.gson:gson", version.ref = "gson" }
```

Comment:
Question: We are using kotlin serializer json in the project instead of Gson. Can we rewrite the gson specifics to kotlinx-json, there should be similar or have an equivalent version.